src/charm.py

```diff
@@ -253,18 +258,16 @@ def _install_apt_packages(self, packages: List[str]) -> None:
             self.unit.status = BlockedStatus("couldn't install MongoDB")
 
     def _initialise_replica_set(self) -> None:
-        """Initialise deployed units as a replica set."""
+        # initilise as a replica set with one replica
```

Comment:
```suggestion
        # initialise as a replica set with one replica
```

Reply:
ah I promise I'll get better at that, the american vs british english is difficult for me 😅